modules/virt-creating-pvc-with-virtctl-guestfs.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-libguestfs-tools-virtctl-guestfs.adoc
+
+[id="virt-creating-pvc-with-virtctl-guestfs_{context}"]
+= Creating a container using virtctl guestfs
+
+You can create a pod with `libguestfs-tools`, mount the PVC, and attach a shell to it:
+
+[source,terminal]
+----
+$ virtctl guestfs -n <namespace> <pvc-name>
+----
+
+NOTE: A <pvc-name> is a required argument. If you do not include the PVC name, an error message appears.
```

Comment:
If you do not include the pvc name,  there is not error message appears.  It will show the messages like this:

$ virtctl guestfs -n  <namespace>
accepts 1 arg(s), received 0

Reply:
I think the documentation is okay on this point.  This is an error message.  I'll grant that it's not a good error message at all but that would not be a documentation bug.  If we want the error message changed then we should file a bug against kubevirt.